go.mod

```diff
@@ -92,4 +89,9 @@ require (
 	sigs.k8s.io/yaml v1.1.0
 )
 
-replace github.com/awslabs/goformation => github.com/errordeveloper/goformation v0.0.0-20190507151947-a31eae35e596
+replace (
+	github.com/awslabs/goformation => github.com/errordeveloper/goformation v0.0.0-20190507151947-a31eae35e596
+	// it appears that we pick wrong version of github.com/docker/distribution, and it breaks k8s.io/kubernetes
```

Comment:
Can you be more specific? The problem is that the version of go releaser we depend on uses a version of docker distribution which is in compatible with the version of k8s.io/kubernetes we use.

Reply:
That's what I said, but sure.

Reply:
To be honest, I'd rather move along and release out, changing this comment will take several minutes as I'll have to rebuild the dependencies image and push it (which takes 10m at least)...